### PR TITLE
🔧 chore: remove em()

### PR DESCRIPTION
**Pull request by ptbrowne**
- Removed em() and font-size()

### CODE REVIEW

File: stylus/components/table.styl [hunk 1]
```diff
@@ -27,8 +27,8 @@ $table
     .coz-table-header
     .coz-table-cell
         box-sizing  border-box
-        padding     em(15px, 14px) em(32px, 14px) // @stylint ignore
-        font-size   14px
+        padding     .9375rem 2rem // @stylint ignore
```
Comment (GoOz) [inline stylus/components/table.styl]: I guess those `@stylint ignore` are useless now.

File: test/basefont.styl [hunk 2]
```diff
@@ -3,9 +3,9 @@ body
 
     p
         font-size 14px
-        padding-top em(1.6em)
-        padding-bottom em(14px)
+        padding-top 22.4rem
```
Comment (GoOz) [inline test/basefont.styl]: huh. Are you sure about this? I guess the math went wild on this one.

File: test/basefont.styl [hunk 3]
```diff
@@ -3,9 +3,9 @@ body
...
+        padding-bottom 1rem
```
Comment (GoOz) [inline test/basefont.styl]: Isn't `1rem`  supposed to be equal to `16px` instead of `14px`?

File: stylus/components/table.styl [hunk 4]
```diff
@@ -56,8 +56,8 @@ $table
         color coolGrey
 
     .coz-table-header
-        padding         em(8px, 12px) em(32px, 12px) // @stylint ignore
-        font-size       12px
+        padding         .5rem 2rem // @stylint ignore
```
Comment (GoOz) [inline stylus/components/table.styl]: Another useless `@Stylint ignore` here

**Comment by ptbrowne** [comment]
Sorry for the typos :/ FYI it's live on https://ptbrowne.github.io/cozy-ui/react/

**Comment by GoOz** [comment]
@ptbrowne still a trailing `@stylint ignore` and we're good to go. :)

**Pull request merged**